chimney/src/main/scala/io/scalaland/chimney/internal/utils/MacroUtils.scala

```diff
@@ -100,6 +101,11 @@ trait MacroUtils extends CompanionUtils {
         // $COVERAGE-ON$
       }
     }
+
+    def coproductSymbol: Symbol = t match {
+      case c.universe.ConstantType(Constant(enumeration: TermSymbol)) => enumeration
```

Comment:
The whole `c.universe` is imported in this scope, seems to be redundant in this pattern.